core/domain/feedback_jobs_one_off_test.py

```diff
@@ -391,6 +391,80 @@ def test_no_cache_update_to_thread_with_anon_then_empty_messages(self):
         self.assertIsNone(model.last_nonempty_message_author_id)
 
 
+class TextMessageLengthAuditOneOffJobTests(test_utils.GenericTestBase):
+    """Tests for the one-off Text message length limit job."""
+
+    # Text with length more than 10000.
+    TEXT = 'a' * 10001
```

Comment:
Can mention the length in the name itself as well to make it more explicit, e.g. "TEXT_WITH_LENGTH_PAST_LIMIT" or something similar.

Reply:
Done.